backend/canisters/user_index/impl/src/queries/users.rs

```diff
@@ -19,13 +19,13 @@ fn users_impl(args: Args, state: &RuntimeState) -> Response {
     let mut current_user: Option<CurrentUserSummary> = None;
 
     if let Some(u) = state.data.users.get_by_principal(&caller) {
-        if let Some(ts) = args
+        if let Some(updated_since) = args
             .user_groups
             .iter()
             .find(|g| g.users.contains(&u.user_id))
             .map(|g| g.updated_since)
         {
-            if u.date_updated > ts || u.date_updated_volatile > ts {
+            if updated_since == 0 || u.date_updated > updated_since || u.date_updated_volatile > updated_since {
```

Comment:
Surely if updated since is 0 then it'll return everything already? Or is the date_updated field 0 for some users?

Reply:
date_updated_volatile defaults to zero

Reply:
But u.date_updated shouldn't be 0. It should at least be the date the user was created otherwise we won't return newly created users

Reply:
This PR is nonsense - fixing it!!